README.md

```diff
@@ -21,6 +21,67 @@ You can deploy Che plugin registry on Openshift with command.
              -p IMAGE_TAG="latest" \
              -p PULL_POLICY="IfNotPresent"
 ```
+## Docker
+```
+docker run -it  -p 8080:8080 eclipse/che-plugin-registry
+```
+
+## Get index list of all plugins
+Example:
+```
+curl  "http://localhost:8080/index.json"
+```
+Response:
+```javascript
+[
+  {
+    "id": "org.eclipse.che.editor.theia",
+    "version": "1.0.0",
+    "type": "Che Editor",
+    "name": "theia-ide",
+    "description": "Eclipse Theia",
+    "links": {
+      "self": "/plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml"
+    }
+  },
+  {
+    "id": "che-service-plugin",
+    "version": "0.0.1",
+    "type": "Che Plugin",
+    "name": "Che Service",
+    "description": "Che Plug-in with Theia plug-in and container definition providing a service",
+    "links": {
+      "self": "/plugins/che-service-plugin/0.0.1/meta.yaml"
+    }
+  },
+  {
+    "id": "che-dummy-plugin",
+    "version": "0.0.1",
+    "type": "Che Plugin",
+    "name": "Che dummy plugin",
+    "description": "A hello world theia plug-in wrapped into a Che Plug-in",
+    "links": {
+      "self": "/plugins/che-dummy-plugin/0.0.1/meta.yaml"
+    }
+  }
+]
+```
+## Get meta.yaml of a plugin
+Example:
+```
+curl  "http://localhost:8080/plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml"
```

Comment:
could it be http://localhost:8080/plugins/org.eclipse.che.editor.theia/1.0.0 ?

`/plugins/<id>/<version>`

Reply:
Not for now. In general   - don't know. Would you like to request this as a new feature?